.github/workflows/build.yml

```diff
@@ -0,0 +1,67 @@
+name: "Build"
+on: push
+env:
+  PREFIX: /opt/tenzir
+  CC: cc
+  CXX: c++
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        configure: ["", "--dev-mode"]
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+          ref: ${{ github.ref }}
+      - uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
+      - name: Install Ubuntu dependencies
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          sudo apt-get -qq update && sudo apt-get -qqy install \
+          gcc-8 g++-8 ninja-build libbenchmark-dev libpcap-dev libssl-dev \
+          libatomic1
+          pip3 install --upgrade pip
+          pip3 install --user cmake
+          ln -sf /usr/local/bin/cmake $(python3 -m site --user-site)/cmake/data/bin/cmake
+      - name: Install macOS dependencies
+        if: matrix.os == 'macos-latest'
+        run: |
+          brew install openssl cmake git ninja python libpcap jq tcpdump rsync pandoc apache-arrow
+      - name: Configure
+        run: |
+          echo "VAST-$(git describe)-$(uname -s)-${{ matrix.configure == '' && 'Release' || 'Debug' }}" > PACKAGE_NAME
+          ./configure --generator=Ninja --package-name=$(cat PACKAGE_NAME) --disable-arrow ${{ matrix.configure }}
```

Comment:
Why aren't the existing CI scripts used here?

Reply:
There are three reasons for this:
- The build matrix is easier to read if the configuration is either empty string or `--dev-mode`. 
- It is easier for a devops person without much knowledge about CMake (i.e., @0ortmann) not having to interact with `CMake -B...` manually.
- We recommend using `configure` in our README, so CI should use it, too.

Reply:
Thanks for the explanation.